install.md

```diff
@@ -1,104 +1,52 @@
 ---
 layout: page
 title: Installing Jupyter
-tagline: Get up and running with JupyterLab or the classic Jupyter Notebook.
+tagline: Get up and running on your computer
 permalink: /install
 ---
 
-## Getting started with JupyterLab
+Project Jupyter's tools are available for installation via the [Python Package Index](https://pypi.org/), the leading repository of software created for the Python programming language. Its offerings can be installed with a variety of package management tools that run via a [command-line interface](https://en.wikipedia.org/wiki/Command-line_interface), including [conda](https://docs.conda.io/), [mamba](https://mamba.readthedocs.io/) and [pipenv](https://pipenv.pypa.io/).
 
-_The [installation guide](http://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html) contains more detailed instructions_
+Which one you use is up to you. Pipenv is recommended by the [Python Packaging Authority](https://www.pypa.io/). Conda and mamba are preferred by many members of [PyData](https://pydata.org/) and can by acquired via [miniforge](https://github.com/conda-forge/miniforge#mambaforge).
```

Comment:
```suggestion
This page uses instructions with [pip](https://pip.pypa.io/en/stable/) as it is [the recommended installation tool for Python](https://packaging.python.org/en/latest/guides/tool-recommendations/#installation-tool-recommendations).
If you require _environment management_ as opposed to just installation, you should look into [conda](https://docs.conda.io/), [mamba](https://mamba.readthedocs.io/), and [pipenv](https://pipenv.pypa.io/).

> **note**: If you install JupyterLab with `conda` or `mamba`, we recommend using [the `conda-forge` channel](https://conda-forge.org/).
```

Reply:
Should the  note go in the Jupyter Lab section, maybe below the install line, rather that in the top section?